File: 1-Iniciante/JavaScript/1015.js
let [x1, y1] = lines.shift().split(" ").map((x) => parseFloat(x));
let [x2, y2] = lines.shift().split(" ").map((x) => parseFloat(x));

let d1 = x2 - x1;
let d2 = y2 - y1;

d1 = Math.pow(d1, 2);
d2 = Math.pow(d2, 2);

let distancia = Math.sqrt(d1 + d2);

console.log(distancia.toFixed(4));

File: 1-Iniciante/Java/1015.java
package beecrowdjava1015;

import java.io.*;
import java.util.*;

public class BeeCrowdJava1015 {

    public static void main(String[] args) throws IOException{
        
        Scanner in = new Scanner(System.in);
      
        double x1, y1, x2, y2, dist;
        x1 = in.nextDouble();
        y1 = in.nextDouble();
        x2 = in.nextDouble();
        y2 = in.nextDouble();
        
        dist = Math.sqrt((Math.pow((x2-x1),2)) + (Math.pow((y2-y1),2)));
        System.out.printf("%.4f\n",dist);
    }
    
}

